### DISCUSSION TITLE
How to save translated page?

### DISCUSSION THREAD

**Discussion by vvkdev** [FilipePS/Traduzir-paginas-web#607] [Q&A]
I've read (i.e. scroll down to the bottom) a long translated page. It looks like fully translated from the top to the bottom.
But when I tried to save this page as pdf in my opera or select all and copy to clipboard, there are some non-translated paragraphs (seems to be random each time of copying).
Can I save it all translated?
Thank you!

**Comment by FilipePS** [comment]
Texts are translated as they appear on your screen. If you scroll too quickly, part of the text will not be translated, because there is a time interval between each translation, if you scroll through a text in that interval it will not be translated.

**Comment by vvkdev** [comment]
Yes, I know it. I scrolled very slow, of course, when I read the page. But how to save the fully translated page after read?

**Reply by FilipePS** [reply]
Can you send me a link to a page where this problem occurs?

**Reply by vvkdev** [reply]
Now , when I reopen the page and slow scroll down, it saves correctly. Don't know, why now it's ok. Sorry for disturbing, I will test further and if the error repeats, I will write. Thank you for the great extension and for the replies!
